src/app.rb

```diff
@@ -39,11 +53,25 @@ def list_all_labels
   end
 
   def list_all_authors
-    puts 'Author Listed'
+    if @authors.empty?
+      puts 'You have No recorded authors yet! '
+      puts
+    else
+      @authors.each do |author|
+        puts "First Name: #{author.first_name}, Last Name: #{author.last_name}"
+      end
+    end
   end
 
   def add_author
-    # Author Properties
+    puts 'Enter Author\'s Id:'
+    id = gets.chomp
```

Comment:
Good work so far with the add_author methods @degisew . 

nevertheless, i would strongly suggest that adding an author id can cause our code to crash in the future as id are supposed to be non-repeatitive. On that basis, i would suggest that in the author class, in the initialize method, id should be set to an auto generator such as 
`id: SecureRandom.hex(7)`

This will create an auto-generated id as String and would avoid any repetition in the future.

Reply:
🚀

Reply:
Hi @uisendy ,
I'm done with the comments. kindly, review my work and let me know if changes are required.